### PR TITLE
EDM-476: apply filters when listing enrollment requests

### PR DESCRIPTION
**Pull request by asafbss**

### CODE REVIEW

File: cmd/flightctl-api/main.go [hunk 1]
```diff
@@ -39,6 +39,12 @@ func main() {
 	}
 	log.Printf("Using config: %s", cfg)
 
+	cfg.Database.Port = uint(8888)
+	cfg.Service.Address = ":3444"
+	cfg.Service.AgentEndpointAddress = ":7445"
+	cfg.Service.AgentGrpcAddress = ":7446"
+	cfg.Queue.AmqpURL = "amqp://user:password@127.0.0.1:5674"
+
```
Comment (fzdarsky) [inline cmd/flightctl-api/main.go:47]: Accidental commit after debugging?

Comment (asafbss) [reply cmd/flightctl-api/main.go]: Yes, my bad. I just reverted this file.
Thanks Frank

**Review by avishayt — CHANGES_REQUESTED** [review]
I think this API would be more friendly if the query param was approved=true/false.  The backend can translate that to status.approval.approved.

**Comment by celdrake** [comment]
> I think this API would be more friendly if the query param was approved=true/false. The backend can translate that to status.approval.approved.

The param change LGTM. However there's an issue ATM, that the current filter only works when obtaining the approved ERs. For the pending ones, no results are returned. I believe this may be caused by https://issues.redhat.com/browse/EDM-484

**Comment by avishayt** [comment]
@asafbss Is this PR still relevant, or will it be changed to use the new query language?

**Comment by asafbss** [comment]
@avishayt the new query language will handle it, I'm closing it.

**Pull request closed without merging**